.circleci/config.yml

```diff
@@ -1,20 +1,25 @@
 version: 2.1
 
 references:
+  - &enable_tags
+    filters:
+      tags:
+        only: 
+          - /.*/
   - &only_tagged
     filters:
       branches:
         ignore:
-        - /^.*$/
+        - /.*/
```

Comment:
won't that ignore all branches… including `master`?

Reply:
Yes, we don't want to deploy on any merges including `master`, only for tags.

Reply:
I suppose it's ok as it should be triggered by the tag, not push to branch
https://circleci.com/docs/2.0/workflows/#executing-workflows-for-a-git-tag

Reply:
Sure, I can change the naming 👍